src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -43,9 +43,10 @@ private void handleCommandEntered() {
 
         try {
             commandExecutor.execute(commandText);
-            commandTextField.setText("");
         } catch (CommandException | ParseException e) {
             setStyleToIndicateCommandFailure();
+        } finally {
+            commandTextField.setText("");
         }
```

Comment:
There is no need to change this, since the "catch" block sets the incorrect command we type to be red. Shifting the commandTextField.setText("") to a "finally" block will erase all inputs regardless of it is right or wrong, then there is no point setting it to red colour (in the catch block already).